packages/localization/src/languages/en.json

```diff
@@ -14,6 +14,7 @@
                 "users": "Manage Users"
             },
             "devices": {
+                "import": "Device import",
```

Comment:
Changes to localization files, should be as a separate commit to this package.

Reply:
done